cpg_pipes/providers/cpg/status.py

```diff
@@ -133,19 +157,31 @@ def add_status_updater_job(
 
         j = b.new_job(job_name, job_attrs)
         j.image(image_path('sm-api'))
+
+        calc_size_cmd = None
+        if output:
+            calc_size_cmd = f"""
+        from cloudpathlib import CloudPath
+        meta['size'] = CloudPath({output}).stat().st_size
+        """
         cmd = dedent(
             f"""\
         cat <<EOT >> update.py
         from sample_metadata.apis import AnalysisApi
         from sample_metadata.models import AnalysisUpdateModel, AnalysisStatus
         from sample_metadata import exceptions
         import traceback
+        
+        meta = dict()
```

Comment:
I think pylint prefers literal dicts over the `dict()` method. If this isn't changing, why not include it directly on the update_analysis_status function invocation?
```suggestion
        meta = {}
```

Reply:
Because it's populated only if `output` is defined

Reply:
> I think pylint prefers literal dicts over the dict() method

`dict()` is better than double-quoting curly braces `{{}}`. I might refactor it though to define the function outside and call it with inspect (or even use Hail's python job)

Reply:
That makes perfect sense - dict function call is perfect